packages/aragon-cli/src/commands/deploy.js

```diff
@@ -101,6 +103,13 @@ exports.task = async ({
 
           task.output = `Deploying '${contractName}' to network`
 
+          let environment = Object.values(module.environments)[0]
```

Comment:
Won't this work only with the first environment?

I think we need to do something like: 

```suggestion
          let environment = Object.values(module.environments)[environment]
```